include/PR/os_host.h

```diff
@@ -0,0 +1,9 @@
+#ifndef PR_OS_HOST_H
+#define PR_OS_HOST_H
+
+void __osInitialize_common(void);
+void __osInitialize_autodetect(void);
+
+#define osInitialize() __osInitialize_common()
```

Comment:
Why does this function have a define? Is this just libultra convention? Is `osInitialize()` used?

Reply:
Yes, it is a libultra convention, it can be set to other functions instead.
https://github.com/decompals/ultralib/blob/main/include/PR/os_host.h#L67